.env.example

```
@@ -1,21 +1,74 @@
 GITHUB_APP_ID=your-github-app-id
+# Multi-line PEM is fine. Or collapse newlines to literal \n for single-line secrets.
 GITHUB_APP_PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----\n...\n-----END RSA PRIVATE KEY-----"
 GITHUB_APP_SLUG=your-github-app-slug
+
+# =============================================================================
+# APP
+# =============================================================================
+# Production URL of the app (no trailing slash). OAuth redirects back here.
+PUBLIC_APP_URL=http://localhost:3000
+# Node adapter binds to HOST:PORT (defaults 0.0.0.0:3000). Override per-env here.
+HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PUBLIC_APP_URL key

(UnorderedKey)